Src/Support/Google.Apis.Auth/OAuth2/Responses/TokenResponse.cs

```diff
@@ -93,6 +93,21 @@ public DateTime Issued
         [Newtonsoft.Json.JsonPropertyAttribute(Order = 2)]
         public DateTime IssuedUtc { get; set; }
 
+        /// <summary>Access token for impersonated credentials.</summary>
+        [Newtonsoft.Json.JsonPropertyAttribute("accessToken")]
+        private string ImpersonatedAccessToken { set => AccessToken = value; }
+
+        /// <summary>ID token for impersonated credentials.</summary>
+        [Newtonsoft.Json.JsonPropertyAttribute("token")]
+        private string ImpersonatedIdToken { set => IdToken = value; }
+
+        /// <summary>
+        /// Access token expiration time for impersonated credentials. It has the RFC3339
+        /// format: "yyyy-MM-dd'T'HH:mm:ss'Z'".
```

Comment:
If this is going to have 9 digits of precision, we should show that in the example.

Reply:
updated the comment